go.sum

```diff
@@ -0,0 +1,118 @@
+github.com/alcortesm/tgz v0.0.0-20161220082320-9c5fe88206d7 h1:uSoVVbwJiQipAclBbw+8quDsfcvFjOpI5iCf4p/cqCs=
```

Comment:
This looks really huge for 4 direct dependencies.
Maybe a `go mod tidy` could help?

Reply:
`go mod tidy` does not help at all with the `go.sum`